main.cpp

```diff
@@ -2,60 +2,78 @@
 
 enum { id_button1 = 1, id_button2 };
 
-void on_create(HWND hw) 
+HBRUSH cyanBrush = nullptr;
+
+void on_create(HWND hw)
 {
-	// TODO: create two child windows of type button
+    CreateWindowW(L"BUTTON", L"Button 1", WS_TABSTOP | WS_VISIBLE | WS_CHILD | BS_DEFPUSHBUTTON,
+        10, 10, 100, 25, hw, (HMENU)id_button1, NULL, NULL);
+
+    CreateWindowW(L"BUTTON", L"Button 2", WS_TABSTOP | WS_VISIBLE | WS_CHILD | BS_DEFPUSHBUTTON,
+        10, 40, 100, 25, hw, (HMENU)id_button2, NULL, NULL);
 }
 
-void on_command(HWND hw, int id) 
+void on_command(HWND hw, int id)
 {
-	// TODO: show message box with text depending on which button was pressed
+    switch (id) {
+    case id_button1:
+        MessageBoxW(hw, L"Button 1 was pressed!", L"Warning", MB_OK | MB_ICONWARNING);
+        break;
+    case id_button2:
+        MessageBoxW(hw, L"Button 2 was pressed!", L"Warning", MB_OK | MB_ICONWARNING);
+        break;
+    }
 }
 
-void on_destroy() 
+void on_destroy()
 {
-	::PostQuitMessage(0);
+    if (cyanBrush) {
+        DeleteObject(cyanBrush);

```

Comment:
Nema potrebe, pogledajte [help za WNDCLASS](https://learn.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-wndclassa): _An application should not delete these brushes._